module/buff/basic_buff.cpp

```diff
@@ -910,68 +931,84 @@ void Detector::calculateTargetPointSet(const float&              _predict_quanti
+  // 计算最终坐标点
+  radio_ =
+      abstract_object::centerDistance(_final_center_r, current_target_.getArmor().getRect().center);
 
   pre_center_.x = radio_ * cos_calcu_ + _final_center_r.x;
   pre_center_.y = radio_ * sin_calcu_ + _final_center_r.y;
 
+  // 计算最终目标的旋转矩形
   target_rect_ = cv::RotatedRect(pre_center_, current_target_.getArmor().getRect().size, 90);
 
+  /* 通过模型计算最终目标点的位置信息（预测点）TODO(fqjun):待优化 */
+  // 计算能量机关的高度
   target_buff_h_ = 800 + sin(final_radian_ - CV_PI) * 800;
   target_y_      = target_buff_h_ + barrel_buff_botton_h_;
 
+  // 计算能量机关目标的直线距离
   final_target_z_ = sqrt((target_y_ * target_y_) + (target_x_ * target_x_));
+  /* 通过模型计算最终目标点的位置信息（预测点） */
 
+  // 保存最终目标的顶点，暂时用的是排序点的返序存入才比较稳定，理论上正确使用应为0123 ！！！
   _target_2d_point.clear();
-
-  _target_2d_point.emplace_back(current_target_.vector2DPoint(0));
-  _target_2d_point.emplace_back(current_target_.vector2DPoint(1));
-  _target_2d_point.emplace_back(current_target_.vector2DPoint(2));
-  _target_2d_point.emplace_back(current_target_.vector2DPoint(3));
-
+  cv::Point2f target_vertex[4];
+  target_rect_.points(target_vertex);
+  _target_2d_point.emplace_back(target_vertex[3]);
+  _target_2d_point.emplace_back(target_vertex[2]);
+  _target_2d_point.emplace_back(target_vertex[1]);
+  _target_2d_point.emplace_back(target_vertex[0]);
+
+#ifdef DEBUG
+  // 最终目标装甲板（预测值）
   for (size_t i = 0; i != 4; ++i) {
-    cv::line(_input_dst_img,
-             _target_2d_point[i],
-             _target_2d_point[(i + 1) % 4],
-             cv::Scalar(0, 130, 255),
-             8);
+    cv::line(_input_dst_img, _target_2d_point[i], _target_2d_point[(i + 1) % 4],
+             cv::Scalar(0, 130, 255), 8);  // orange
   }
 
-  cv::circle(_input_dst_img, _final_center_r, radio_, cv::Scalar(0, 255, 125), 2, 8, 0);
-  cv::circle(_input_dst_img, pre_center_, 3, cv::Scalar(255, 0, 0), 3, 8, 0);
+  cv::circle(_input_dst_img, _final_center_r, radio_, cv::Scalar(0, 255, 125), 2, 8, 0);  // 轨迹圆
+  cv::circle(_input_dst_img, pre_center_, 3, cv::Scalar(255, 0, 0), 3, 8, 0);  // 预测值的中点
 
-  cv::line(_input_dst_img, pre_center_, _final_center_r, cv::Scalar(0, 255, 255), 2);
-  cv::line(_input_dst_img, current_target_.getArmor().getRect().center, _final_center_r, cv::Scalar(0, 255, 0), 2);
+  cv::line(_input_dst_img, pre_center_, _final_center_r, cv::Scalar(0, 255, 255),
+           2);  // 预测点和圆心的连线
+  cv::line(_input_dst_img, current_target_.getArmor().getRect().center, _final_center_r,
+           cv::Scalar(0, 255, 0), 2);  // 装甲板和圆心的连线
 
+  // 顺时针表示顶点顺序,红黄蓝绿
   cv::circle(_input_dst_img, _target_2d_point[0], 10, cv::Scalar(0, 0, 255), -1, 8, 0);
   cv::circle(_input_dst_img, _target_2d_point[1], 10, cv::Scalar(0, 255, 255), -1, 8, 0);
   cv::circle(_input_dst_img, _target_2d_point[2], 10, cv::Scalar(255, 0, 0), -1, 8, 0);
   cv::circle(_input_dst_img, _target_2d_point[3], 10, cv::Scalar(0, 255, 0), -1, 8, 0);
+#endif  // DEBUG
```

Comment:
```suggestion
#endif  // DEBUG
```
对比之前结束 RELEASE 的标记 !RELEASE

Reply:
这个是什么意思？